Babylon/utils/environment.py

```diff
@@ -305,7 +319,7 @@ def get_project_secret(self, organization_id: str, workspace_key: str, name: str
             logger.error(e)
             sys.exit(1)
         prefix = f'{self.organization_name}/{self.tenant_id}/projects/{self.context_id}'
-        schema = f'{prefix}/{self.environ_id}/{organization_id}/{workspace_key}/{name}'
+        schema = f'{prefix}/{self.environ_id}/{organization_id.lower()}/{workspace_key.lower()}/{name.lower()}'
```

Comment:
Same as before for this one: the lower() may be applied to the entire string definition